ember-headless-form/src/components/-private/field.hbs

```diff
@@ -1,4 +1,10 @@
-{{#let (unique-id) (unique-id) (fn @set @name) as |fieldId errorId setValue|}}
+{{#let
+  (unique-id)
```

Comment:
I missed this from before..but why are there two `unique-id`s here?

Reply:
See how they are used `as` different variables basically: `fieldId` and `errorId`. `fieldId` is the id of the control (input), which `<label for="<fieldId>">` references, and `errorId` is the id of the element holding all error messages, which `<input aria-errormessage="<errorId>">` references.

Probably we could rename `fieldId` to maybe `controlId`, but it doesn't really matter - at least for users - as that is used only internally...

Reply:
TIL